graylog2-server/src/main/java/org/graylog2/shared/bindings/providers/OkHttpClientProvider.java

```diff
@@ -97,8 +105,36 @@ public void connectFailed(URI uri, SocketAddress sa, IOException ioe) {
             };
 
             clientBuilder.proxySelector(proxySelector);
+
+            if (!isNullOrEmpty(httpProxyUri.getUserInfo())) {
+                final List<String> list = Splitter.on(":")
+                        .limit(2)
+                        .omitEmptyStrings()
+                        .splitToList(httpProxyUri.getUserInfo());
```

Comment:
Technically, the "user" and the "password" components of the HTTP/1.1 user info can be empty but neither variant is supported if we use this `Splitter` configuration.

Do we want to support these edge cases? They're arguably valid, the question is if we want to support them.